spec/message.abnf

```diff
@@ -3,7 +3,7 @@ message = simple-message / complex-message
 simple-message = [simple-start pattern]
 simple-start = simple-start-char / text-escape / placeholder
 pattern = *(text-char / text-escape / placeholder)
-placeholder = expression
+placeholder = expression / markup
```

Comment:
I realize this is cosmetic (or academic?), but I think I was assuming we'd instead do:

```abnf
pattern = *(text-char / text-escape / placeholder / markup)
placeholder = expression
```

That is, that markup won't end up being placeholders, but rather a new type of pattern elements, similar to how you implemented it in the data model. I presume that ABNF will have the most impact on shaping MF2's vocabulary, which is why I think we should be careful here.

Thoughts?

Reply:
This part of the ABNF was explicitly included in the design doc, and so I'd really rather not change that here. I also think that having a catch-all name for "stuff in a pattern that has curly braces" is useful, and that "placeholder" works for that.

I would strongly prefer discussing this change as a separate follow-up to this PR.